pkg/api/norman/customization/aks/handler.go

```diff
@@ -40,6 +40,7 @@ type Capabilities struct {
 	ClientID         string `json:"clientId"`
 	ClientSecret     string `json:"clientSecret"`
 	ResourceLocation string `json:"region"`
+	CurrentVersion   string `json:"current_version"`
```

Comment:
This should probably be camel case to be consistent with the other parameters (e.g. `clientSecret`)